### PR TITLE
css cleaup

### PR DESCRIPTION
**Pull request by BO41**
some css changes regarding style.
I hope I didn't break anything, but the css files really look terrible.
Hopefully this will help you port everything to scss. But I guess remaking the css would be easier since finding out which rule does what and if it still is used seem impossible. Also it is not recommended to use so many ID selectors but rather use class selectors.
If you don't merge, please consider at least removing empty rule sets and make a consistent formatting. 

please review carefully and consider improving the css at least a little bit if this is too much

#88

### CODE REVIEW

File: css/faq.css [hunk 1]
```diff
@@ -159,143 +191,152 @@ h3 > a > i.fa-chevron-left {
...
+
+.tiles-container.faq-tiles .tile>.tile-head>strong {
     padding: 7px 15px;
     width: calc(100% - 45px);
 }
-.tiles-container.faq-tiles .tile > .tile-body {
+
+.tiles-container.faq-tiles .tile>.tile-body {
     display: none;
-    #height: 0;
```
Comment (TobiGr) [inline css/faq.css]: Rules which have a hash in front of them will not be applied and thus can be removed.

Comment (BO41) [reply css/faq.css]: ah, okay.
I was not sure if this is a typo or some old browser hack.
Will remove them :+1:

**Comment by BO41** [comment]
@TobiGr that's a problem. So would you rather like all media queries where they were or can I group them all together at the bottom of the style sheet?

Looking what broke is not a reliable method. So I would rather revert the change or as mentioned put all queries at the bottom, this should not break anything

File: css/faq.css (hunk 1 above)
Comment (TheAssassin) [reply css/faq.css]: Rules with a hash are of course _tried_ to be interpreted. However, this is invalid syntax in CSS, and therefore the line will be ignored. There is no commenting syntax in CSS.

**Comment by TobiGr** [comment]
Then please keep order as it has been before. As I said, I want to port everything to SASS/SCSS and the current structure makes this a lot easier.

**Comment by BO41** [comment]
@TobiGr okay, will change it back (or make a fresh PR).

I am not sure about the current structure.
But it does not seem like you are having a default display target.
Like making the page mobile first and then change it with media queries to bigger displays.
(If you are not already doing this, just a suggestion to make it easier :)

**Comment by TobiGr** [comment]
I did this for most elements, but there are some (e.g. the feature slides or the main page header) which I mainly developed for desktops and then thought about a nice design for mobiles. But yes this was planned to be done with the SCCS refactor too.

**Comment by BO41** [comment]
replaced with #91 which is better

**Pull request closed without merging**